### PR TITLE
feat(releasing): New task to automatically create release schedule

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
This new task create the [release page and the associated notes](https://datadoghq.atlassian.net/wiki/spaces/agent/pages/3752427616/Release+1.2.3).
A potential improvement would be to create the google calendar entries
as well.
For now to be launched locally with atlassian credentials, could be added as a workflow on github

### Motivation
Automation

### Additional Notes
cc @kacper-murzyn

### CODE REVIEW

File: tasks/libs/releasing/documentation.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    password = os.environ['ATLASSIAN_PASSWORD']
+    auth = requests.auth.HTTPBasicAuth(username, password)
+    space_key = "agent"
+    parent_page_id = "2244936127"
+    # Make the POST request to create the page
+    domain = "https://datadoghq.atlassian.net/wiki"
+    url = f"{domain}/rest/api/content"
+    headers = {"Content-Type": "application/json"}
+    page_title = f"Agent {version}"
+    teams = get_releasing_teams()
```
Comment (pgimalac) [inline tasks/libs/releasing/documentation.py:20]: What do you think of sorting teams ? The generated page is pretty inconvenient to fill right now 😄 
```suggestion
    teams = sorted(get_releasing_teams())
```

Comment (chouetz) [reply tasks/libs/releasing/documentation.py:20]: I think I agree!

Comment (CelianR) [reply tasks/libs/releasing/documentation.py:20]: Maybe you can sort in the `get_releasing_teams` function

File: tasks/release.py [hunk 2]
```diff
@@ -1800,3 +1801,22 @@ def generate_release_metrics(ctx, milestone, freeze_date, release_date):
...
+@task
+def create_schedule(_, version, freeze_date):
+    """
+    Create confluence pages for the release schedule.
+    freeze_date - date when the code freeze was started. Expected format YYYY-MM-DD, like '2022-02-01'
+    """
+    required_environment_variables = ["ATLASSIAN_USERNAME", "ATLASSIAN_PASSWORD"]
+    if not all(key in os.environ for key in required_environment_variables):
+        raise Exit(
+            color_message(
```
Comment (CelianR) [inline tasks/release.py:1815]: **❓ question:** ‏Is it ok to use color messages within errors ?

**Review by CelianR — APPROVED** [review]

**Review by FlorentClarret — APPROVED** [review]
LGTM, I agree with the other comments. It would be nice to have some tests if possible

**Pull request merged**